From fd506da4cb2d050e1e7a82dfc31e93c1dd284209 Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Fri, 20 Mar 2020 10:54:54 +0800
Subject: [PATCH 501/767] misc: mrvl-loki: Drop the unused variable in
 mrvl_loki_probe()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix the below build warning:
drivers/misc/mrvl-loki.c: In function ‘mrvl_loki_probe’:
drivers/misc/mrvl-loki.c:49:11: warning: unused variable ‘regval’ [-Wunused-variable]
   49 |  uint64_t regval;
      |           ^~~~~~

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/misc/mrvl-loki.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/misc/mrvl-loki.c b/drivers/misc/mrvl-loki.c
index cf956e3e1949..9a21cc158240 100644
--- a/drivers/misc/mrvl-loki.c
+++ b/drivers/misc/mrvl-loki.c
@@ -46,7 +46,6 @@ static int mrvl_loki_probe(struct platform_device *pdev)
 	struct mrvl_loki *ml;
 	struct device *dev = &pdev->dev;
 	struct pci_dev *bphy_pdev;
-	uint64_t regval;
 	int ret = 0;
 
 	ml = devm_kzalloc(dev, sizeof(*ml), GFP_KERNEL);
-- 
2.31.1

